### DISCUSSION TITLE
posts/2023/09/five-years-in/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#33] [Announcements]
# posts/2023/09/five-years-in/

There’s now been a half-decade of whatever it is I do here.

https://www.brycewray.com/posts/2023/09/five-years-in/

**Comment by davidsneighbour** [comment]
I shall comment on all your future posts with "Shouldn't you be finishing your novel instead of writing this?" Congrats on 5 years.

**Reply by brycewray** [reply]
@davidsneighbour Thanks!

As for the novel: my last serious effort at it was in 2020, and it didn’t last long. Long story (no pun intended — well, not really, anyway).
